composer/yamls/hwbenchmark/bert.yaml

```diff
@@ -0,0 +1,83 @@
+# Use a bert-base model, initialized from scratch
```

Comment:
This YAML looks almost identical to https://github.com/mosaicml/composer/blob/dev/composer/yamls/models/bert-base.yaml.  Let's not duplicate.